Essentials/src/config.yml

```diff
@@ -484,6 +484,19 @@ send-fly-enable-on-join: true
 # Give someone permission to teleport to a world with essentials.time.world.<worldname>.
 world-time-permissions: false
 
+# Specify cooldown for both Essentials commands and external commands as well.
+# All commands do not start with a Forward Slash (/). Instead of /msg, write msg
+#
+# Wildcards are supported. E.g.
+# - '*i*': 50
+# adds a 50 second cooldown to all commands that include the letter i
+#command-cooldowns:
+#  feed: 100 # 100 second cooldown on /feed command
+#  '*': 5 # 5 Second cooldown on all commands
+
+# Whether command cooldowns should be persistent past server shutdowns
+#command-cooldown-persistence: false
```

Comment:
Eventually each cooldown should have the ability to be persistent, but for now I made it into a configurable variable just so that it's an option for whoever needs it. I'm thinking something close to mini game servers.

I can set it to true by default.